VideoLocker/src/main/java/org/edx/mobile/interfaces/TextResourceProvider.java

```diff
@@ -10,4 +10,10 @@
      * @return The text resource
      */
     int getTextResource();
+    /**
+     * Get the server mapping string value.
```

Comment:
I think the method can be named `getQueryValue()` or `getQueryParamValue()`. The description can be "Get the value of the query parameter". The method should not be defined here though, as this is a dedicated interface for serving the display text resource.